### PR TITLE
add missing file check function

### PR DESCRIPTION
**Pull request by axif0**
### Contributor checklist

- [X] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [X] I have tested my code with the `pytest` command as directed in the [testing section of the contributing guide](https://github.com/scribe-org/Scribe-Data/blob/main/CONTRIBUTING.md#testing)

---

### Description
Added `check_missing_query_files` function in `check_project_structure.py` github action to check all missing `data-type` query for all available `language`.

### Related issue

- #357

### CODE REVIEW

**Comment by axif0** [comment]
@andrewtavis we have these missing query files to add.
image

File: src/scribe_data/check/check_project_structure.py [hunk 1]
```diff
@@ -73,7 +73,32 @@
 BASE_DIR = "../language_data_extraction"
 
 
-def check_data_type_folders(path, language, subdir, errors):
+def check_for_sparql_files(folder_path, data_type, language, subdir, missing_queries):
+    """
+    Check if a data-type folder contains at least one .sparql file.
+
+    Args:
```
Comment (andrewtavis) [inline src/scribe_data/check/check_project_structure.py:80]: @axif0 Can you please remember to format your docstrings in a way that's consistent with the rest of the project. As it's written this won't be parsed into the documentation.

**Review by andrewtavis — APPROVED** [review]
Ok this is really good, @axif0 :) Thanks for it. Please do note the docstrings in the future as with that this review would be pretty effortless. As you can see I commented the checks out for now as we're not close to being able to not have them commented, but we can get to it later 😊

File: src/scribe_data/check/check_project_structure.py (hunk 1 above)
Comment (axif0) [reply src/scribe_data/check/check_project_structure.py]: Sorry for this. And thank you. I will remember this in future.

**Pull request merged**